### PR TITLE
List DarkPay (D4RK)

### PR DESCRIPTION
**Pull request by ghost**
- Official project URL: https://darkpaycoin.io
- Official block explorer URL: https://explorer.darkpay.market

Thanks !

### CODE REVIEW

File: assets/src/main/java/bisq/asset/coins/DarkPay.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+package bisq.asset.coins;
+
+import bisq.asset.Base58BitcoinAddressValidator;
+import bisq.asset.Coin;
+import bisq.asset.NetworkParametersAdapter;
+
+public class DarkPay extends Coin {
+    public DarkPay() {
+        super("DarkPay", "D4RK", new Base58BitcoinAddressValidator(new DarkPayMainNetParams()), Network.MAINNET);
```
Comment (blabno) [inline assets/src/main/java/bisq/asset/coins/DarkPay.java]: `Network.MAINNET` is default so remove that param.

**Review by blabno — CHANGES_REQUESTED** [review]
NACK per comments

File: assets/src/main/java/bisq/asset/coins/DarkPay.java (hunk 1 above)
Comment (ghost) [reply assets/src/main/java/bisq/asset/coins/DarkPay.java]: Hello, thanks for info, done :
https://github.com/bisq-network/bisq/pull/2918/commits/0a6c4c7cad5d7b154fe6c012a7f980c48bbd61cc

**Comment by blabno** [comment]
Squash into a single commit.

**Comment by ghost** [comment]
Tempted an update from main bisq repo before squash, don't know if it was a good idea...

**Comment by blabno** [comment]
You have touched way to many files than needed for listing a single asset.

**Comment by ghost** [comment]
Hello, i did a full reset to be even with Bisq master branch, then added the commit again, should be good now.

**Review by blabno — CHANGES_REQUESTED** [review]
NACK lack of code formatting.

**Comment by ghost** [comment]
> NACK lack of code formatting.

Hi, fixed and squashed.

**Review by blabno — APPROVED** [review]
ACK

**Pull request merged**